src/main/scala/loamstream/loam/intake/aggregator/Metadata.scala

```diff
@@ -1,35 +1,200 @@
 package loamstream.loam.intake.aggregator
 
+import scala.util.Try
+
+import com.typesafe.config.Config
+
+import Metadata.Defaults
+import loamstream.conf.ConfigParser
+import scala.util.Success
+import loamstream.util.Tries
+import loamstream.util.Options
+
 /**
  * @author clint
  * Feb 11, 2020
  */
 final case class Metadata(
```

Comment:
I'm thinking it'd be wise to have (if easily possible) some kind of "generic" k/v "extra" field in `Metadata`. As-is, if Jason, Marcin, whoever wanted to add some extra bit of info (e.g. a link to a paper), that would require someone to come into this code, add an instance variable, update the code that writes it out, etc. 

Obviously that requires something similar in the dig-aggregator-intake python code as well that can read it and write it out to S3. Maybe just a "comments" key that allows for arbitrary k/v (string) pairs? Thoughts?

Reply:
That's a decent idea, I'll see what I can do.

Reply:
This turned out to be more complicated than expected.  It's not _too_ bad, and is a decent idea.  I'll revisit it in another PR.